go/libraries/doltcore/table/untyped/sqlexport/sqlwriter.go

```diff
@@ -145,6 +160,98 @@ func (w *SqlExportWriter) insertStatementForRow(r row.Row) (string, error) {
 	return b.String(), nil
 }
 
+func (w *SqlExportWriter) WriteDeleteRow(ctx context.Context, r row.Row) error {
```

Comment:
You should update sqlwriter_test.go with simple tests for the new methods

Reply:
done